src/Model/Request/SubModel/Head/External/Tracking.php

```diff
@@ -43,12 +41,50 @@ class Tracking extends AbstractModel
      */
     public $admittedFields = [
         'Id' => [
-            'mandatory' => false,
-            'cdata' => true,
+            'mandatory' => true,
+            'multiple' => true,
+            'instanceOf' => 'Head\\External\\Tracking\\Id',
```

Comment:
Adding 'instanceOf' does not make it backwards compatible. If you test this with the current version of the library the below code works and you can dump and see that the ID is set. but if you switch to your changes you get an error.
"Fatal error:  Uncaught Model exception : Wrong class instance set to 'Id' - 'RatePAY\Model\Request\SubModel\Head\External\Tracking\Id"


$mbArr = new RatePAY\ModelBuilder('Head');
$mbArr->setArray([
    'SystemId' => "Example",
    'Credential' => [
        'ProfileId' => PROFILE_ID,
        'Securitycode' => SECURITYCODE
    ],
    'Meta' => [
        'Systems' => [
            'System' => [
                'Name' => 'Magento 2'
            ]
        ]
    ],
    'External' => [
        'Tracking' => [
            'Id'=> '234',
       ]
    ]
]);

Reply:
Well, this should be changed by the implementation.
I added the backward compatibility over a little hack.